modules/vba_documents/app/mailers/vba_documents/monthly_report_mailer.rb

```diff
@@ -18,10 +18,12 @@ def self.fetch_recipients
 
     # rubocop:disable Metrics/MethodLength
     # rubocop:disable Metrics/ParameterLists
-    def build(monthly_totals, summary, still_processing, avg_processing_time, last_month_start, last_month_end)
+    def build(monthly_totals, summary, still_processing, still_success,
```

Comment:
Looks like there's an extra newline here

Reply:
that line break is there to fix the linter line length error